website/docs/community/contributing/contributing-online-community.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Online community building"
+id: "contributing-online-community"
+---
+
+### Maintaining a channel in the dbt Community Slack
+
+#### What is it
+
+The dbt Slack is the place for real time conversations with the dbt Community. Slack channels exist to for specific locations, tools, industries and methodologies. In order to make sure that every channel has dedicated attention from a committed community member, we have Community Maintainers who oversee the discussion in particular channels.
+
+#### Contribution opportunities
+  
+We’re working on compiling a list of existing channels that could benefit from additional community maintainers. In the meantime, feel free to be on the lookout for opportunities to help maintain new channels that are getting spun up by keeping an eye on #channel-requests in the dbt slack.
+
+#### Sample contributions
+
+- Karen Hsieh's [contributions](https://getdbt.slack.com/archives/C02TU2DSKND/p1661483529756289) to the #local-taipei channel are a fantastic example to learn from.
+
+#### How to get started
+
+- Read the guide to [Maintaining a Slack Channel](https://docs.getdbt.com/community/maintaining-a-channel)
+- Jump in to threads in #channel-requests for new channels and offer to help maintain or mention that you are interested in helping maintain a channel and we’ll reach out.
+
+### Participating on Discourse
+
+#### What is it
+[GitHub Discussions](https://github.com/dbt-labs/docs.getdbt.com/discussions) are the dbt community’s platform for long-lived discussions about dbt, analytics engineering, and analytics. They’re a place for us to build up a long-lasting knowledge base around the common challenges, opportunities, and patterns we work with every day.
+
+#### How is Discourse different from other community platforms?
+
+- Compared to threads in the dbt Community Slack, GitHub Discussions are more asynchronous, long lived and intentional. Conversations on Slack are great for quick back and forths, but are hard to reference over time and not as good at building long-lasting community knowledge.
+- Compared to drafting a blog post for the Developer Blog, GitHub Discussions have a faster iteration loop, require no direct intervention from the dbt Labs team and are more focused around interacting with others’ views on a given topic or question. A good Discussion can be focused around asking a question you’d like feedback on or it can be sharing an idea that you’re not quite ready to turn into a fully fledged blog post.
```

Comment:
```SUGGESTION
- Compared to drafting a blog post for the Developer Blog, the Community Forum has a faster iteration loop, require no direct intervention from the dbt Labs team and are more focused around interacting with others’ views on a given topic or question. A good Discussion can be focused around asking a question you’d like feedback on or it can be sharing an idea that you’re not quite ready to turn into a fully fledged blog post.